--- a.py	2023-07-21 23:24:13.603148653 +0200
+++ b.py	2023-07-21 23:28:12.145941183 +0200
@@ -1,13 +1,6 @@
 """Fava - A web interface for Beancount."""
 from __future__ import annotations
 
-from contextlib import suppress
-from importlib.metadata import PackageNotFoundError
-from importlib.metadata import version
-
-with suppress(PackageNotFoundError):
-    __version__ = version(__name__)
-
 LOCALES = [
     "bg",
     "ca",
